### ISSUE TITLE
CutOut in Albumentations

### ISSUE THREAD

**Issue by ternaus** [uoguelph-mlrg/Cutout#19]
We added a generalized version of Cutout in [Albumentains](https://albumentations.ai/) called [CourseDropout](https://albumentations.ai/docs/api_reference/augmentations/dropout/coarse_dropout/#albumentations.augmentations.dropout.coarse_dropout.CoarseDropout)

Generalized as it allows:
1. Masking not one but many regions with the number of holes that is sampled or fixed.
2. Shape could be square as in Cutout, or recnatgular
3. Could be applied not only to images, but `images`, `segmentation masks` and `key points`

Feel free to check [documentation](https://albumentations.ai/docs/api_reference/augmentations/dropout/coarse_dropout/#albumentations.augmentations.dropout.coarse_dropout)

**Comment by gwtaylor** [comment]
Thanks for sharing. Is there something going on with your site's DNS? 
I tried resolving albumentations.ai through multiple DNS servers and I'm not getting a hit.

**Comment by ternaus** [comment]
yes, issues with DNS, working on it, will message let you know here when I will fix it.

**Comment by ternaus** [comment]
@gwtaylor Fixed

**Comment by gwtaylor** [comment]
Thanks, it works now!

**Comment by ternaus** [comment]
@gwtaylor 
If you, as an expert in the field of Computer Vision and data augmentations have any questions / proposals / feedback or just open for a short video chat, feel free to email me at: iglovikov@gmail.com, or message at LinkedIn at https://www.linkedin.com/in/iglovikov/
